### DISCUSSION TITLE
Add a security vulnerabilities page on quarkus website

### DISCUSSION THREAD

**Discussion by farnulfo** [quarkusio/quarkus#27026] [Community]
Hi,
On quarkus blog we can see release with security fixes like https://quarkus.io/blog/quarkus-2-10-3-final-released/ .
But I missed a permanent security section with quarkus releases about security.
Apache tomcat security vulnerabilities for a major version like https://tomcat.apache.org/security-9.html is really a great model that I would like to have for quarkus.

What do you think ?

**Comment by maxandersen** [comment]
interesting idea - lets make this an issue/feature request.
